install-k6.js

```diff
@@ -5,6 +5,7 @@ const K6_VERSION = 'v0.55.0'
 const K6_PATH_MAC_AMD = `k6-${K6_VERSION}-macos-amd64`
 const K6_PATH_MAC_ARM = `k6-${K6_VERSION}-macos-arm64`
 const K6_PATH_WIN_AMD = `k6-${K6_VERSION}-windows-amd64`
+const K6_PATH_LINUX_AMD = `k6-${K6_VERSION}-linux-amd64`
```

Comment:
If we're only supporting amd64 for now, maybe we should abort the installation when `process.arch` isn't supported.

Reply:
I would say, let's add it :)

Reply:
I presume this is no longer accurate as we should be able to support both now?

Reply:
Correct we support both